packages/notifi-frontend-client/lib/models/FusionEvent.ts

```diff
@@ -0,0 +1,61 @@
+import { Types } from '@notifi-network/notifi-graphql';
+
+export type FusionEventMetadata = {
+  uiConfigOverride?: {
+    topicDisplayName?: string;
+    historyDisplayName?: string;
+    icon?: Types.GenericEventIconHint;
+    customIconUrl?: string;
```

Comment:
Add two optional two fields:
1. icon 
2. customIconUrl
The `icon` is currently in `Filter`, but it seems to be more intuitive to have it in `uiConfigOverride`. 
Need BE's help to migrate it (`icon`) and also add a new field `cutomIconUrl?` in `GenericEventDetails`
CC: @kai-notifi @nimesh-notifi @marukohao

Reply:
Yeah, I like that.
If both are defined, what's the behavior? Try URL and then fallback to icon?
icon?: Types.GenericEventIconHint;
customIconUrl?: string;